modules/module-postgres-storage/src/storage/batch/PostgresBucketBatch.ts

```diff
@@ -529,31 +529,28 @@ export class PostgresBucketBatch
       const current_data_lookup = new Map<string, CurrentDataDecoded>();
       for await (const currentDataRows of db.streamRows<CurrentData>({
         statement: /* sql */ `
-          WITH
```

Comment:
This makes the query a bit easier to read, doesn't really have much of an impact on sync performance.